packages/plugin-javascript/index.js

```diff
@@ -20,7 +20,7 @@ function getTopModuleName(target) {
 
 export function javascriptFile({ path, target }) {
   const list = [];
-  const extName = ['.js', '.jsx', '.ts', '.tsx', '.ls', '.json'];
+  const extName = ['.js', '.jsx', '.ts', '.tsx', '.ls', '.json', '.vue'];
```

Comment:
Adding `.vue` here is not need unless you want to support such a import `import item.vue`. I never worked with Vue so not sure if this is a common case or not.

Reply:
Reverting that change also makes the tests passing, since the other change is not covered by any tests which isn't a big deal in my opinion.

Reply:
Right, I'll update that, I kinda misunderstood your comment at the issue.

Reply:
My fault, I thought that's needed to support Vue.